lib/file_path_parser.go

```diff
@@ -0,0 +1,48 @@
+package lib
+
+import (
+	"go/build"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+func fileFileForTest(suite *Suite, test *Test, pkg *build.Package, strip string) string {
```

Comment:
Not sure I like the name `fileFile` prefix. Hard to understand what this does from the name, maybe a document?